app.js

```diff
@@ -0,0 +1,60 @@
+'use strict';
+var express = require('express');
+var app = express();
+var trucks = require('./trucks');
+var bodyparser = require('body-parser');
+
+app.use(bodyparser.urlencoded({ extended: true}));
+app.use(bodyparser.json());
+app.use(express.static('public'));
+
+app.get('/trucks', function (request, response) {
+  response.json(trucks.getTrucks());
+});
+// the 'name' parameter will be available on the request.params object
+app.get('/trucks/:name', function (request, response) {
+  var truckName = request.params.name;
+  var truck = trucks.getTruck(truckName);
+  if(truck) {
+    response.json(truck);
+  }
+  else {
+    response.status(404).json('Truck not found');
+  }
+});
+//All foodtypes
+app.get('/food-types', function (request, response) {
+  var foodList = trucks.getFoodTypes();
+  var foodListHTML = '<h1>All Food Types</h1>';
+  foodList.forEach(function (foodType) {
+    foodListHTML += '<li><a href="/food-types/' + foodType + '/">' + foodType + '</a></li>';
```

Comment:
Nice - you have this stuff down pretty good - I'd take a look into doing templates in Express for the next step here.